### PR TITLE
[RW-5799][risk=no] Align main CPU/ram/disk to grid

### PR DESCRIPTION
**Pull request by calbach**
An improvement - though unfortunately I couldn't get the two grids to align properly, let me know if you have ideas on this.

image

### CODE REVIEW

File: ui/src/app/pages/analysis/runtime-panel.tsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ const styles = reactStyles({
   },
   formGrid: {
     display: 'grid',
-    gridTemplateColumns: '1fr 1fr 1fr 1fr 3rem 1fr',
```
Comment (calbach) [inline ui/src/app/pages/analysis/runtime-panel.tsx:46]: I didn't understand the 3rem in there - let me know if that was needed for some reason.

Comment (petesantos) [reply ui/src/app/pages/analysis/runtime-panel.tsx:46]: I was seeing Disk(GB) wrap iirc.

**Review by petesantos — APPROVED** [review]
Is the issue too much spacing between the label and the input?
We could make the grid 3 columns and have each label/input combo in one column

**Comment by calbach** [comment]
The minor issue I mentioned in the PR description is that the two grids are not aligned to one another, which would be ideal. I don't see a way of doing that without hardcoding the column widths, which I'd rather not do.

**Pull request merged**